bin/cask

```diff
@@ -153,20 +166,36 @@ def format_version(version):
     return '.'.join(str(part) for part in version)
 
 
+def get_missing_packages():
+    """Get a report of the missing direct dependencies.
+
+    The report is a human-readable list of the any direct dependencies which are
+    required but not present. The empty string is returned if there are none.
+
+    """
+    process = subprocess.Popen([sys.executable, CASK, "missing"], stdout=subprocess.PIPE)
```

Comment:
Please check the return code of the process and raise an exception if it's not zero.  We should fail when we fail to get the list of packages, instead of silently printing something.

Furthermore, I do not like that this method returns a string, let alone a "human-readable" string.  It should return a list of missing packages, or—probably more appropriate in our case—a boolean indicating whether packages are missing or not (though in this case the function should be called `are_packages_missing` or so).